lib/commands/general.js

```diff
@@ -31,11 +31,6 @@ commands.closeApp = async function closeApp () {
   throw new errors.UnsupportedOperationError('Please quit the session in order to close the application under test');
 };
 
-// eslint-disable-next-line require-await
-commands.reset = async function reset () {
```

Comment:
I thought reset has been disabled for espresso.

Reply:
Yes, I thought so, too.
Let's keep this line since we thought the same thing (it was disabled).